.github/workflows/cd.yml

```diff
@@ -391,44 +391,65 @@ jobs:
           name: dist-artifacts
           path: /tmp/dist-artifacts
 
+      - name: Prepare "latest" artifacts
+        run: |
+          cp -r /tmp/dist-artifacts /tmp/dist-artifacts-latest
+          # Replace the version with "latest" in the zip file name
+          find /tmp/dist-artifacts-latest -type f -exec bash -c 'mv "$1" "${1//${{ fromJSON(needs.ci.outputs.plugin).version }}/latest}"' _ {} \;
+        shell: bash
+
       - name: Login to Google Cloud
         uses: google-github-actions/auth@v2.1.8
         with:
           workload_identity_provider: "projects/304398677251/locations/global/workloadIdentityPools/github/providers/github-provider"
           service_account: github-plugin-ci-workflows@grafanalabs-workload-identity.iam.gserviceaccount.com
 
       - name: Determine GCS artifacts paths
+        id: paths
         run: |
           plugin_version='${{ fromJSON(needs.ci.outputs.plugin).version }}'
           gcs_artifacts_release_base="${{ env.GCS_ARTIFACTS_BUCKET }}/${{ fromJSON(needs.ci.outputs.plugin).id }}/release"
           {
+            echo "gcs_artifacts_release_base=$gcs_artifacts_release_base"
             echo "gcs_artifacts_release_path_latest=$gcs_artifacts_release_base/latest"
             echo "gcs_artifacts_release_path_tag=$gcs_artifacts_release_base/$plugin_version"
             fn=$(basename ${{ needs.ci.outputs.gcs-universal-zip-url-commit }})
+            latest_fn=${fn//${{ fromJSON(needs.ci.outputs.plugin).version }}/latest}
             if [ "${{ matrix.platform }}" == 'any' ]; then
               echo "gcs_upload_glob=$fn*"
+              echo "gcs_upload_glob_latest=$latest_fn*"
             else
               # strip the extension (.zip) and append the platform instead
               echo "gcs_upload_glob=${fn%.*}.${{ matrix.platform }}*"
+              echo "gcs_upload_glob_latest=${latest_fn%.*}.${{ matrix.platform }}*"
             fi
-          } >> "$GITHUB_ENV"
+          } >> "$GITHUB_OUTPUT"
```

Comment:
The switch from `$GITHUB_ENV` to `$GITHUB_OUTPUT` is not strictly related to this PR, but it's better because those variables don't have to be shared with all steps